docs/learn.md

```diff
@@ -0,0 +1,112 @@
+# Creation Of Tkinter-Designer
+
+## Introduction
+
+ Hello everyone, I'm Parth Jadhav. I am a 17 year old student. And I would like to share how I created this repository.
+
+## How does it work? 
+ Tkinter Designer uses Figma to generate the GUI. Users design the desired user interface in Figma. Then paste the URL into Tkinter Designer. TkinterDesigner uses the Figma API to get information about the file. Use that data to generate Tkinter-based GUI code in Python, using templates and more.
+
+## How it all started
+
+So, about a year ago, I took this [online course](https://www.udemy.com/course/100-days-of-code) for Python on Udemy. I was new to Python. I was having fun with Python until it came to creating GUIs.
+
+I didn't find it interesting or easy enough to create GUIs in Python. They were mostly dull and boring. And it took a lot of effort to make it look good. So I decided to make something which will make GUIs beautiful and easy.
+
+## Discovery & Search
+
+If you have a problem, first search to see if anyone has solved it. Often times the problem you are trying to solve is already solved by someone. In that case you could either use that solution or build up on that solution or create a solution of your own. I chose to create my own solution to the problem.
+
+I went on to search for any libraries or softwares which can solve this problem. And I found a repository -> [PySimpleGUI](https://github.com/PySimpleGUI/PySimpleGUI). I tried it and it was amazing. But I wanted to Automate stuff and do something fun. I didn't have an concrete idea on what and how to build. But wanted to create something which would make creating GUIs easier in Python.
+
+One day I was just surfing on Youtube and found this guy working with [Figma API](https://www.figma.com/developers/api). Figma is a web based graphic design software. And it's API was able to provide the data about the design file the user is working on.
+
+That gave me the idea to use Figma as the external tool for this project.
+
+## First steps
+
+Starting a project can sometimes be overwhelming. Sometimes we don't know where to start. Sometimes we are afraid of failures.Sometimes we don't know what to do next. It happens with everyone of us. The key is to just take that first step and start. It's not always easy to start. But it's always worth it.
+
+I went on to create `main.py` file. First thing I did was to write code to get Data from [FigmaAPI](https://www.figma.com/developers/api) and Print it. I now had a code which would get the data from Figma about a file that User designed.
```

Comment:
Could you add a code snippet of the code you used to get Data from the FigmaAPI and explain it. This would be helpful for a new user.

Reply:
Sure thing, I'll add it.

Reply:
Hey @ericdrosado , It's present on line 43.